scripts/waffles.coffee

```diff
@@ -11,37 +12,59 @@ waffleTypes = ['chocolate', 'cheese', 'kaya', 'peanut', 'blueberry', 'plain']
 module.exports = (robot) ->
   timeoutId = null
 
+  # produces a summary of current orders
   getOrders = () ->
     output = 'Orders so far:\n'
     for waffleType in waffleTypes
-      numType = robot.brain.get(waffleType)
-      output += "#{waffleType}: #{numType}\n" if numType != 0
+      nameList = robot.brain.get(waffleType)
+      numType = nameList.length
+      names = getNames nameList
+      output += "#{waffleType}: #{numType} #{names}\n" if numType != 0
     timeoutId = null
     output
 
+  # produces a nice comma separated string of names surrounded by parantheses
+  getNames = (nameList) ->
+    names = '('
+    for name in nameList
+      names += "#{name}, "
+    names = names[0..-3]
+    names += ')'
+
+  # produces a nice string of available flavours
+  getAvailableFlavours = () ->
+    available = 'Available flavours are '
+    for waffleType in waffleTypes
+      available += "#{waffleType}, "
+    available[0..-3]
+
+  # returns true if the waffles? command was issued within the last 15 minutes
+  # false otherwise
   isOrderActive = () ->
     waffleTime = robot.brain.get('waffleTime')
     now = new Date()
-    #
+
     # if it's within 15 minutes
     if (now - 15 * 60 * 1000) < waffleTime
       true
     else
       false
 
+  # listen out for waffles? to start consolidating
   robot.hear /waffles\?/i, (msg) ->
```

Comment:
If you want `waffles?`to be a command, you should get the bot to respond to it rather than hear it.